<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unlockWgt</class>
 <widget class="QWidget" name="unlockWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font:12px &quot;微软雅黑&quot;;
}
QPushButton{
font:12px &quot;微软雅黑&quot;;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background:#eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="passwdLabel">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
font-size:12px;
color:#868989;
line-height:12px;
text-align:right;</string>
          </property>
          <property name="text">
           <string>密码：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwdLineEdit">
          <property name="minimumSize">
           <size>
            <width>218</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>218</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#passwdLineEdit{
background:#ffffff;
border:1px solid #c3caca;
border-radius:2px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:1px;
text-align:justify;
	padding-left:10px;
}
#passwdLineEdit:hover{
border:1px solid blue;
}
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>65</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#confirmBtn{
background:#ffd500;
border-radius:2px;
}
#confirmBtn:pressed{
margin:1px;
}
</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#cancelBtn{
background:#ffffff;
border-radius:2px;
}
#cancelBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
font-size:12px;
color:#868989;
line-height:12px;
text-align:right;</string>
          </property>
          <property name="text">
           <string>用户名：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="usernameHintLabel">
          <property name="minimumSize">
           <size>
            <width>218</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>218</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left:10px;
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#323434;
letter-spacing:1px;
text-align:justify;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="hintLabel">
        <property name="styleSheet">
         <string notr="true">#hintLabel{
color: rgb(255, 0, 0);
padding-left:65px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
